src/server/routes/consents.ts

```diff
@@ -333,6 +334,7 @@ router.post(
     let state = res.locals;
 
     const sc_gu_u = req.cookies.SC_GU_U;
+    const _cmpConsentedState = req.body._cmpConsentedState;
 
```

Comment:
Curious for my own understanding, how is the `__cmpConsentedState` first initialised and added to the request?

Reply:
It's value is submitted as an invisible form input on every post req whe the user hits "continue"